### ISSUE TITLE
Kudos skeleton size mismatch

### ISSUE THREAD

**Issue by mkimberlin** [objectcomputing/check-ins#2605]
**Describe the bug**
Kudos skeletons are small for a large width screen.

**To Reproduce**
1. On a desktop with a reasonably large screen/resolution, Go to ADMIN
2. Click on Kudos
3. Click on either the submitted or approved tabs and see the skeleton on load

**Expected behavior**
Skeletons should try to match the general size and layout of the item(s) they are representing.

**Screenshots**
Watch carefully...they flash by quickly!
https://github.com/user-attachments/assets/0aeb12d9-9620-4152-b08a-b4aabca71465

**Comment by mkimberlin** [comment]
This still exists, if someone wants to pick it up while Will is out.

<img width="1411" alt="Screenshot 2024-10-03 at 7 11 39 PM" src="https://github.com/user-attachments/assets/419d769a-ec86-4680-bb0e-0a31af3643ad">

**Comment by thelenw** [comment]
@mkimberlin I fixed this in #2603 which you merged with the develop branch. See #2607 and #2603. I fixed this issue and reported as much.

**Comment by thelenw** [comment]
This is resolved.

**Comment by mkimberlin** [comment]
@thelenw This issue still was recreated because the size was not correct after those changes.

**Comment by thelenw** [comment]
My understanding was that the fix for the skeleton loader flashing was to remove that by loading the page components before the skeleton of the menu was displayed unnecessarily. Am I missing something here? On the develop branch as of last night, when you load the 'Manage Kudos' page, no skeleton of the menu is displayed and all page elements seem to be appropriately sized.

**Comment by mkimberlin** [comment]
> My understanding was that the fix for the skeleton loader flashing was to remove that by loading the page components before the skeleton of the menu was displayed unnecessarily. Am I missing something here? On the develop branch as of last night, when you load the 'Manage Kudos' page, no skeleton of the menu is displayed and all page elements seem to be appropriately sized. @mkimberlin 

This had nothing to do with the menu. It's the skeletons of the kudos that are displayed during load. You can see them flash quickly in the video. They need to be widened to fit the screen.

**Comment by mkimberlin** [comment]
To be clear @thelenw the problem isn't the flashing... It's their size. They should show if the load takes a while. You can force that with chrome developer tools bandwidth limiting.

**Comment by thelenw** [comment]
And to clarify, you merged the update with the fix for the flashing menu skeleton to the developer branch. Have you experienced excessive load times on the Manage Kudos page, or on any other page? The skeleton menu glitch was present across the Check-Ins site before the changes that I wrote and you merged. I would opt for no specific handling for excessively slow connections, above a specific warning message--either would be more appropriate than the menu loader because for the load time to take that long, the User's connection would have to be so bad that they should be aware of it.

**Comment by thelenw** [comment]
@mkimberlin and I resolved this in person. The break down is:

* Further to the timer mechanism I enacted to fix the ```skeleton loader``` (referred to as menu above) flashing issue, I did not believe further changes to the ```skeleton loader``` were necessary.
* Michael explained that in an edge case where the User has slow connection, we need the ```skeleton loader``` to display until the page loads, AS a mechanism to inform of their slow connection. This was the disconnect.
* My task now is to fix the ```skeleton loader``` to display appropriately in case of poor connection. Both changes mentioned above--fix ```skeleton loader``` flashing and use it as slow connection warning--are important and this is a productive resolution.
